### PR TITLE
DE1255 - fixing group connector issues

### PR DESCRIPTION
**Pull request by IngageMaddoxT**

### CODE REVIEW

File: CI/SQL/20160301132600_US3354-GroupConnector.sql [hunk 1]
```diff
@@ -7,21 +7,42 @@ GO
...
+	CONSTRAINT [FK_GroupConnector_Project] FOREIGN KEY ([Project_ID]) REFERENCES [cr_Projects]([Project_ID]),  
+	CONSTRAINT [FK_GroupConnector_CreatedBy] FOREIGN KEY ([Created_By]) REFERENCES [cr_Registrations]([Registration_ID]),
 	CONSTRAINT [FK_GroupConnector_Domains] FOREIGN KEY ([Domain_ID]) REFERENCES [dp_Domains]([Domain_ID]),
 	CONSTRAINT [PK_GroupConnector] PRIMARY KEY CLUSTERED([GroupConnector_ID] ASC)
 )
-END
+
+
+
+-- DOWN
```
Comment (andycanterbury) [inline CI/SQL/20160301132600_US3354-GroupConnector.sql]: Commented code

Comment (IngageMaddoxT) [inline CI/SQL/20160301132600_US3354-GroupConnector.sql]: that's the tear down script.  Mr. Brewer requiring that...

File: CI/SQL/20160301140100_US3354-GroupConnector-Page.sql [hunk 2]
```diff
@@ -27,14 +39,22 @@ BEGIN
...
+IF NOT EXISTS (SELECT 1 FROM [dbo].[dp_Page_Section_Pages] WHERE Page_Section_ID = 21 AND Page_ID = @PAGE_ID)
+BEGIN
+INSERT INTO [dbo].[dp_Page_Section_Pages]
+           ([Page_ID]
+           ,[Page_Section_ID])
+     VALUES
+           (@PAGE_ID, @PAGE_SECTION_ID)
+END
+
+INSERT INTO dp_Role_Pages (Role_ID, Page_ID, Access_Level) VALUES (@ROLE_ID, @PAGE_ID, @ACCESS_LEVEL_FULL)
```
Comment (andycanterbury) [inline CI/SQL/20160301140100_US3354-GroupConnector-Page.sql]: Are we scripting security? Thought that was up to Mike?

File: CI/SQL/20160309105101_US3354-groupconnector-registrants-sub-page.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    ,@SELECTED_RECORD_EXPRESSION
+    ,@FILTER_KEY
+    ,@Relation_Type_ManyToMany
+    ,@DISPLAY_COPY
+);
+
+SET IDENTITY_INSERT [dbo].[dp_Sub_Pages] OFF;
+
+
+INSERT INTO dp_Role_Sub_Pages (Role_ID, Sub_Page_ID, Access_Level) VALUES (@ROLE_ID, @SUB_PAGE_ID, @ACCESS_LEVEL_FULL);
```
Comment (andycanterbury) [inline CI/SQL/20160309105101_US3354-groupconnector-registrants-sub-page.sql]: Again, are we scripting security for these?

**Pull request merged**